### ISSUE TITLE
How to setup HTTP API for checking Progress for AWS EC2 Ubuntu?

### ISSUE THREAD

**Issue by ParveenBhadooOfficial** [xmrig/xmrig#2436]
https://xmrig.com/docs/miner/api doesn't provide much info on how to setup everything correctly.

**Comment by Spudz76** [comment]
Use site `http://workers.xmrig.info/` it is a fully locally-running API client.

You can also inspect the code to find out how to use the API (or set up your own internal workers-site) using the [source](https://github.com/xmrig/xmrig-workers/)

**Comment by colorfulcvn** [comment]
> Use site [workers.xmrig.info](http://workers.xmrig.info/) it is a fully locally-running (in your browser) API client.
> 
> You can also inspect the code to find out how to use the API (or set up your own internal workers-site) using the [source](https://github.com/xmrig/xmrig-workers/)

Can it support decives behind NAT?

**Comment by Spudz76** [comment]
Yeah if your browser is running within the LAN behind the NAT (it runs completely in browser).

**Comment by jiangjinhua2007** [comment]
Unsubscribe




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xmrig/xmrig"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月11日(星期五) 晚上10:31
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [xmrig/xmrig] How to setup HTTP API for checking Progress for AWS EC2 Ubuntu? (#2436)





 
Yeah if your browser is running within the LAN behind the NAT (it runs completely in browser).
